### PR TITLE
Muon segments in MiniAOD (92X version)

### PR DESCRIPTION
**Pull request by arizzi**
This should be equivalent of (merged) #19488 for 92X.

The very same branch is used for the two PRs.

### CODE REVIEW

**Comment by slava77** [comment]
why a backport for miniAOD?
Is it fixing a bug?
If not, please close and wait for T0 etc to advance to 93X.

**Comment by arizzi** [comment]
this is meant for miniAOD coming with MC production, if 92X is going to be used for MiniAOD production of MC samples.

**Comment by slava77** [comment]
the plan since months was that 2017 miniAOD goes together with the MC v2. I'd like to keep it this way until some XC meeting overrides it.

**Comment by slava77** [comment]
On 8/3/17 7:18 AM, arizzi wrote:
> that's fine with me, but in today PPD we were asked about the status of
> miniAOD in 927 as groups may need it.
> As far as I'm concerned we can make MC v1 with miniAOD v0 (= whatever is
> right now in 927) but frankly backporting the few PRs that only entered
> 93X so far is quite cheap and doesn't interfer with anything (just add
> some features in miniaod wrt miniaod currently produced at Tier0).

it interferes with prompt reco
==> to resolve, a re-miniAOD campaign has to be defined for data so that
we can have a consistent set of products

 this PR changes the content of existing product
same for #20038.

perhaps this discussion better move to emails/HN


tomorrow OPS may be a good place to decide.

File: PhysicsTools/PatAlgos/python/slimming/slimmedMuons_cfi.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
     slimCaloVars = cms.string("1"),
     slimKinkVars = cms.string("1"),
     slimCaloMETCorr = cms.string("1"),
-    slimMatches = cms.string("0"),
+    slimMatches = cms.string("1"),
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/slimming/slimmedMuons_cfi.py]: this changes existing parts of the slimmedMuons and falls outside the scope of 92X, which I think is now limited only to plain additions.

**Comment by arizzi** [comment]
I do not think miniAOD has any such "only additions" rule. Where/when was
it discussed?

**Comment by slava77** [comment]
On 8/3/17 11:06 AM, arizzi wrote:
> I do not think miniAOD has any such "only additions" rule. Where/when was
> it discussed?

we can discuss tomorrow in the OPS.

this PR will apply do prompt data taking
there is no plan to redo miniAOD for all preceding data in 92X.
The primary policy is that produced products and their content should be
consistent for all data used for production.
exceptions cover pure additions.

**Comment by arizzi** [comment]
well, the slimming of the matches just reduce the precision with reduce mantissa, it doesn't change the datamembers and doesn't remove anything.
It is no different than a change of threshold in a global tag ending up producing less noise clusters (which is definitely allowed)

**Comment by slava77** [comment]
-1

as decided in Friday OPS, this is not required for MCv1 [CMSSW_9_2_8]

**Pull request closed without merging**